sdk/digitaltwins/digitaltwins/review/digitaltwins.api.md

```diff
@@ -0,0 +1,354 @@
+## API Report File for "@azure/digitaltwins"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import * as coreHttp from '@azure/core-http';
+import { OperationOptions } from '@azure/core-http';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { PageSettings } from '@azure/core-paging';
+import { RequestOptionsBase } from '@azure/core-http';
+import { RestResponse } from '@azure/core-http';
+import { TokenCredential } from '@azure/core-http';
+
+// @public
+export interface DigitalTwinModelsAddOptionalParams extends coreHttp.OperationOptions {
+    models?: any[];
+}
+
+// @public
+export type DigitalTwinModelsAddResponse = ModelData[] & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: ModelData[];
+    };
+};
+
+// @public
+export type DigitalTwinModelsGetByIdResponse = ModelData & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: ModelData;
+    };
+};
+
+// @public
+export interface DigitalTwinModelsListOptionalParams extends coreHttp.OperationOptions {
+    dependenciesFor?: string[];
+    digitalTwinModelsListOptions?: DigitalTwinModelsListOptions;
+    includeModelDefinition?: boolean;
+}
+
+// @public
+export interface DigitalTwinModelsListOptions {
+    maxItemCount?: number;
+}
+
+// @public
+export type DigitalTwinModelsListResponse = PagedModelDataCollection & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: PagedModelDataCollection;
+    };
+};
+
+// @public
+export interface DigitalTwinsAddHeaders {
+    etag?: string;
+}
+
+// @public
+export interface DigitalTwinsAddRelationshipHeaders {
+    etag?: string;
+}
+
+// @public
+export interface DigitalTwinsAddRelationshipOptionalParams extends coreHttp.OperationOptions {
+    relationship?: any;
+}
+
+// @public
+export type DigitalTwinsAddRelationshipResponse = DigitalTwinsAddRelationshipHeaders & {
+    body: any;
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: any;
+        parsedHeaders: DigitalTwinsAddRelationshipHeaders;
+    };
+};
+
+// @public
+export type DigitalTwinsAddResponse = DigitalTwinsAddHeaders & {
+    body: any;
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: any;
+        parsedHeaders: DigitalTwinsAddHeaders;
+    };
+};
+
+// @public
+export class DigitalTwinsClient {
+    // Warning: (ae-forgotten-export) The symbol "AzureDigitalTwinsAPIOptionalParams" needs to be exported by the entry point index.d.ts
```

Comment:
forgotten export

Reply:
Fixed.

---
In reply to: [470199113](https://github.com/Azure/azure-sdk-for-js/pull/10560#discussion_r470199113) [](ancestors = 470199113)